sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/persons/PersonDirectorySteps.java

```diff
@@ -64,38 +71,136 @@ public PersonDirectorySteps(
           webDriverHelpers.isElementVisibleWithTimeout(UUID_INPUT, 20);
         });
 
-    When(
-        "I navigate to the last created Person page via URL",
+    Then(
+        "I choose random value for Year of birth filter",
```

Comment:
add also the location of the action since this filter is available in multiple places in the application